app/controllers/amp_articles_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AmpArticlesController < ActionController::Base
+  include NotFound
```

Comment:
This controller is exposed to `Cross-Site Request Forgery` according to hakiri because the method`protect_from_forgery` is not used.